### PR TITLE
changed VPC address to addresses

### PR DESCRIPTION
**Pull request by sawyerawesome**
## Types of changes

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes if appropriate.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by sawyerawesome** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "gaia",
    "pr-id": "817",
    "commit-sha": "1e2a053c22c7704a689d38509c019c69fb280cae",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "backend",
    "pr-id": "1774",
    "commit-sha": "66ffca1b43bc1c9537cd0062148cab01fbcb7e63",
    "pipeline": "master"
  }
]
```

**Comment by sawyerawesome** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "1774",
    "commit-sha": "37caee467d6ba7a72c8480cf6948e9ceab734c68",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "817",
    "commit-sha": "1e2a053c22c7704a689d38509c019c69fb280cae",
    "pipeline": "master"
  }
]
```

**Comment by sawyerawesome** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "gaia",
    "pr-id": "817",
    "commit-sha": "1e2a053c22c7704a689d38509c019c69fb280cae",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "backend",
    "pr-id": "1774",
    "commit-sha": "4ea5590f335397365e513ca78580acc4fc216159",
    "pipeline": "master"
  }
]
```

**Comment by sawyerawesome** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "1774",
    "commit-sha": "a82dd5ef8e5f328dfd9e9ccaa33ef10145b0e460",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "817",
    "commit-sha": "1e2a053c22c7704a689d38509c019c69fb280cae",
    "pipeline": "master"
  }
]
```

**Comment by sawyerawesome** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "1774",
    "commit-sha": "0414b42e424c09794bacd1a2023e45b979682007",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "817",
    "commit-sha": "1e2a053c22c7704a689d38509c019c69fb280cae",
    "pipeline": "master"
  }
]
```

**Comment by sawyerawesome** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "1774",
    "commit-sha": "c826c17a304e4d97f08e84da560dae8c7af0f42c",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "817",
    "commit-sha": "1e2a053c22c7704a689d38509c019c69fb280cae",
    "pipeline": "master"
  }
]
```

**Comment by sawyerawesome** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "1774",
    "commit-sha": "c826c17a304e4d97f08e84da560dae8c7af0f42c",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "817",
    "commit-sha": "1e2a053c22c7704a689d38509c019c69fb280cae",
    "pipeline": "master"
  }
]
```

**Comment by sawyerawesome** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "1774",
    "commit-sha": "215e964d8a65f7832765b5b051a8e5a5bf82e233",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "817",
    "commit-sha": "700cf040dd528acad0027d6ff0430ecdaa5be832",
    "pipeline": "master"
  }
]
```

**Comment by sawyerawesome** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "1774",
    "commit-sha": "5a4d0613c6f5731d223c45e97c81d118aabda105",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "817",
    "commit-sha": "700cf040dd528acad0027d6ff0430ecdaa5be832",
    "pipeline": "master"
  }
]
```

File: specs/+cloudvpcdata.spec [hunk 1]
```diff
@@ -11,12 +11,15 @@ model:
 # Attributes
 attributes:
   v1:
-  - name: address
-    description: Address CIDR of the VPC.
-    type: string
+  - name: addresses
```
Comment (t00f) [inline specs/+cloudvpcdata.spec:14]: What is happening for existing data? Do we need a migration or something?

**Review by jping0929 — APPROVED** [review]

**Pull request merged**